pom.xml

```diff
@@ -117,10 +117,12 @@
     <dependency>
       <groupId>com.fasterxml.jackson.core</groupId>
       <artifactId>jackson-core</artifactId>
+      <version>2.9.0</version>
```

Comment:
Shouldn't we update the BOM (`vertx-dependencies`) rather than this POM?

Reply:
yes it will be, before merging that PR, I'll update the vertx-dependencies, for now it's hardcoded here.

And it's not a BOM :-)